addons/fastroping/CfgWeapons.hpp

```diff
@@ -0,0 +1,60 @@
+class CfgWeapons {
+    class InventoryItem_Base_F;
+	class ACE_ItemCore;
```

Comment:
You need to convert existing tab characters, your editor probably has a command to do this